_scripts/instruction-widget/templates/getting-started/renewal.html

```diff
@@ -37,4 +32,12 @@
 revocation happened for some reason). Please select a random minute within the
 hour for your renewal tasks.</p>
 </aside>
-{{/cron_included}}
+
+<p>An example cron job might look like this, which will run at noon and midnight every day:</p>
+
+<pre>0 0,12 * * * {{base_command}} renew {{#cbauto}}--noself-upgrade{{/cbauto}}</pre>
```

Comment:
I think by doing this there will be some fraction of users who just take this command as is. Additionally, we tell the user above to randomize the time it runs but we don't do it ourselves. I think we should change the command run to something like:
```
python -c 'import random; import time; time.sleep(random.random() * 60)' && {{base_command}} ...
```
I care less about this, but it might not hurt to pick a time other than 0 and 12 since that's what Debian and Ubuntu uses so the load on Let's Encrypt is further distributed. If you change this, don't forget to update the sentence about what this crontab does above.

Finally, the flag should be `--no-self-upgrade`.